src/cosmic_ray/cli.py

```diff
@@ -173,7 +173,7 @@ def handle_baseline(args):
 
     # Copy input work-item, but tell it to use the no-op operator. Create a new
     # session containing only this work-item and execute this new session.
-    with use_db(baseline_session_file, mode=WorkDB.Mode.create) as db:
+    with use_db(mode=WorkDB.Mode.create) as db:
```

Comment:
I would prefer an option to keep the baseline database around for post-mortem analysis. In fact, I'd like that to be the default behavior, but we could have a flag to `baseline` that says to delete the database on exit.

Reply:
Reversed